src/Sylius/Behat/Service/ApiSecurityService.php

```diff
@@ -39,6 +39,13 @@ public function __construct(AbstractBrowser $client, SharedStorageInterface $sha
 
     public function logIn(UserInterface $user): void
     {
+        /**
+         * Hot-fix: Added since conflict with 'lcobucci/jwt' in version 3.4
+         * Problem occurs when in calls is using 'trigger_error' with error level E_USER_DEPRECATED
+         * Then run code is interrupt and error is throwing in up
+         */
+        error_reporting(~E_USER_DEPRECATED);
```

Comment:
This will change the global state of the application - I'd rather see it inside some bootstrap file just for the test environment.

Reply:
I can add config to our behat.yaml : 
`calls:`
      `error_reporting: 16383`
Then error doesn't crash our test but all error stack trace is visible: 
<img width="607" alt="Screenshot 2020-11-26 at 20 24 29" src="https://user-images.githubusercontent.com/39232096/100386566-0672a800-3026-11eb-9020-b2d1c882a318.png">

This problem is known in the past:
https://stackoverflow.com/questions/31050716/how-to-run-behat-tests-when-there-are-errors-of-level-e-user-deprecated